lib/tests/it/fixture.rs

```diff
@@ -58,18 +69,18 @@ impl Fixture {
         TESTREF
     }
 
+    #[context("Updating test repo")]
     pub(crate) fn update(&mut self) -> Result<()> {
         let repopath = &self.srcdir.join("repo");
         let repotmp = &repopath.join("tmp");
         let srcpath = &repotmp.join("exampleos-v1.tar.zst");
         std::fs::write(srcpath, EXAMPLEOS_V1)?;
         let srcpath = srcpath.as_str();
-        let repopath = repopath.as_str();
         let testref = TESTREF;
-        bash!(
-            "ostree --repo={repopath} commit -b {testref} --no-bindings --tree=tar={srcpath}",
+        bash_in!(
+            self.dir.open_dir("src")?,
+            "ostree --repo=repo commit -b ${testref} --no-bindings --tree=tar=${srcpath}",
```

Comment:
I am confused on where is repo declared here. I see the tests with `--repo=repo` too but Based on te line you removed I would expect some variable here too?

Reply:
Good question!  The change here is because we use `bash_in!` which changes the working directory for the bash code, we can just use relative paths there now.

Reply:
Ahh makes sense.